src/App.jsx

```diff
@@ -14,57 +15,67 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGoods, setValue] = useState('Jam');
```

Comment:
Using a string to manage the state of `selectedGoods` can lead to potential bugs if the goods list changes or if there are duplicate names. Consider using an index or an object to manage the state more robustly.